MekHQ/docs/history.txt

```diff
@@ -21,6 +21,7 @@ MEKHQ VERSION HISTORY:
 + PR #5228: Added Monthly StratCon Support Points Generation #5228
 + Fix #5181: Generate required lances for manual AtB contracts for StratCon init
 + PR #5234: Refined Turnover Prompt Dialog Text
++ Fix #5239: Fixed Simulated Relationship Behavior
```

Comment:
So this is something we handle _when_ the PR is merged. So it shouldn't be done as part of the PR. When we do a bulk merge, having multiple PRs affecting single lines in history.txt will just create a pile of conflicts and will make everyone sad.

Reply:
That took a few tries to resolve properly, but it's fixed I think.